src/maintagent/agent/process.c

```diff
@@ -24,81 +24,102 @@
 #include "maintagent.h"
 
 /**********  Globals  *************/
-extern PGconn    *pgConn;        ///< database connection
+PGconn* pgConn = NULL;        // the connection to Database
```

Comment:
Can you please retain the `///<` format for comment string? It is used by Doxygen to create documentation comments for global variables.